automatic/kubescape/update.ps1

```diff
@@ -0,0 +1,35 @@
+[CmdletBinding()]
+param([switch] $Force)
+
+Import-Module AU
+
+function global:au_BeforeUpdate {
+  Get-RemoteFiles -Purge -NoSuffix
+}
+
+function global:au_SearchReplace {
+  @{
+    ".\legal\VERIFICATION.txt" = @{
+      "(?i)(^\s*64\-bit software.*)\<.*\>" = "`${1}<$($Latest.URL64)>"
+      "(?i)(^\s*checksum\s*type\:).*"      = "`${1} $($Latest.ChecksumType64)"
+      "(?i)(^\s*checksum(64)?\:).*"        = "`${1} $($Latest.Checksum64)"
+    }
+  }
+}
+
+function global:au_GetLatest {
+  $LatestRelease = Get-GitHubRelease kubescape kubescape
+
+  $checksumAsset = $LatestRelease.assets | Where-Object {$_.name -eq 'kubescape-windows-latest.sha256'} | Select-Object -ExpandProperty browser_download_url
+  $checksum64 = Invoke-WebRequest -Uri $checksumAsset -UseBasicParsing
+
+  return @{
+    Version        = $LatestRelease.tag_name.TrimStart("v")
+    URL64          = $LatestRelease.assets | Where-Object {$_.name -eq 'kubescape.exe'} | Select-Object -ExpandProperty browser_download_url
+    ReleaseURL     = $LatestRelease.html_url
```

Comment:
You are passing this but not using it? In the `au_searchReplace` function above, can you add an update to the `<releaseNotes>` field in the .nuspec file, or remove this entirely as it's not going to be used?

Reply:
I will just remove this as they actually don't have a dedicated place for release Notes.